packages/application-config/src/formatters.ts

```diff
@@ -55,6 +56,22 @@ const formatEntryPointUriPathToResourceAccessKey = (
     })
     .join('');
 
+/**
+ * The function formats the `permissionName` to a resource access key.
+ * It makes the first character of the string and the next character after a special character an uppercase.
+ *
+ * @example
+ * - avengers --> Avengers
+ * - the-avengers --> TheAvengers
+ * - avengers-01 --> Avengers01 [ℹ️ mind the difference here compared to `formatEntryPointUriPathToResourceAccessKey`] */
+const formatPermissionNameToResourceAccessKey = (permissionName: string) =>
```

Comment:
```suggestion
const formatPermissionGroupNameToResourceAccessKey = (permissionName: string) =>
```

Reply:
Thanks, I assume that if we want to change it here I need to refactor it everywhere to keep it consistent (`permissionName` -> `groupName`)

Reply:
Good point, maybe it's better yes so that we have less confusion to the already confused naming formats 😅